client/src/assets/index.js

```diff
@@ -13,6 +13,9 @@ import EVENT_11 from './events-page/past-events/22_23/11.svg';
 import EVENT_12 from './events-page/past-events/22_23/12.svg';
 import EVENT_13 from './events-page/past-events/22_23/13.svg';
 import EVENT_14 from './events-page/past-events/22_23/14.svg';
+import EVENT_15 from './events-page/past-events/23_24/1.svg';
```

Comment:
I think it's better to separate 23-24's past events with 22-23's past events. Meaning, add a prefix before the image number (i.e., EVENT_22_23_1, EVENT_22_23_2). Use **find and replace** instead of doing it manually. After doing this, `EVENT_15` should be `EVENT_23_24_1`.